lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -22,6 +48,17 @@
         Logger.log('edx.certificate.shared', data);
       });
     });
+
+    function shareOnFacebook(text) {
+        FB.ui({
+            method: 'feed',
+            name: text,
+            link: "${request.build_absolute_uri(reverse('cert_html_view', kwargs=dict(user_id=str(user.id),course_id=unicode(course_id))))}",
+            picture: "${request.build_absolute_uri(course_image_url)}",
+            caption: 'Caption like which appear as title of the dialog box', // TODO# need to update as per requirement
+            description: 'Small description of the post' // TODO# need to update as per requirement
+        });
```

Comment:
@asadiqbal08 @pbaruah is there a sandbox or a way to view how these strings would appear in context before we suggest wording? Thanks!